rf-notebook/README.md

```diff
@@ -0,0 +1,32 @@
+# RasterFrames Jupyter Notebook Docker Image
+
+RasterFrames provides a Docker image with a Jupyter Notebook pre-configured with RasterFrames in the PySpark kernel.
+
+## Launching RasterFrames Notebook from Source
+
+First, have Docker installed.
+
+Then, from the source root directory, run
+
+```bash
+sbt rf-notebook/startRFNotebook
```

Comment:
A couple of thoughts.

1) Is it possible to have the commands be a little simpler `start` vice `startRFNotebook`?
2) What do you think about reinstating previous readme's user-facing discussion on  running the image from docker hub, but perhaps in the docs rather than in the readme here? Do we intend to still push to docker hub?
3) Is it possible to have a convenience to run the built image? (lower priority)

Reply:
1. Yes; 
2. Will do, once I publish and updated version (the name is singular, so repo needs to be changed)
3. That's what `startRFNotebook` does... so maybe I'm misinterpreting the question.

Reply:
# 3 I meant run the image without re-building. I say lower prioirity because I imagine the sbt pattern is more for developers

Reply:
@vpipkt That's what `quickStartRFNotebook` does. Let me know what's a better name.